orca-slicer 2.2.0 build fails on master due to missing webkit2gtk-4.1

## Describe the bug

Currently, orca-slicer build fails on master:

```console
error: builder for '/nix/store/1q4nrwy5zr4xkpyaxkg88jnw1kmfyhib-orca-slicer-2.2.0.drv' failed with exit code 1;
       last 25 log lines:
       > -- Looking for XOpenDisplay in /nix/store/qrj5f1mb08k5jg7y9ikzb1njli0gvmxz-libX11-1.8.10/lib/libX11.so;/nix/store/hpd1dss7lmrywlr37vncyzbhzp6rgri5-libXext-1.3.6/lib/libXext.so
       > -- Looking for XOpenDisplay in /nix/store/qrj5f1mb08k5jg7y9ikzb1njli0gvmxz-libX11-1.8.10/lib/libX11.so;/nix/store/hpd1dss7lmrywlr37vncyzbhzp6rgri5-libXext-1.3.6/lib/libXext.so - found
       > -- Looking for gethostbyname
       > -- Looking for gethostbyname - found
       > -- Looking for connect
       > -- Looking for connect - found
       > -- Looking for remove
       > -- Looking for remove - found
       > -- Looking for shmat
       > -- Looking for shmat - found
       > -- Looking for IceConnectionNumber in ICE
       > -- Looking for IceConnectionNumber in ICE - found
       > -- Checking for module 'webkit2gtk-4.1'
       > --   No package 'webkit2gtk-4.1' found
       > CMake Error at /nix/store/4kb60dy0dg80ww8gh2av4by4vwn8l48k-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:645 (message):
       >   The following required packages were not found:
       >
       >    - webkit2gtk-4.1
       >
       > Call Stack (most recent call first):
       >   /nix/store/4kb60dy0dg80ww8gh2av4by4vwn8l48k-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:873 (_pkg_check_modules_internal)
       >   src/slic3r/CMakeLists.txt:666 (pkg_check_modules)
       >
       >
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/1q4nrwy5zr4xkpyaxkg88jnw1kmfyhib-orca-slicer-2.2.0.drv'.
```

## Steps To Reproduce

Steps to reproduce the behavior:

1. use master branch as your nixpkgs input
2. add `orca-slicer` to your packages
3. `nixos-rebuild`

## Expected behavior

I expect the build to succeed.

## Screenshots

<!-- If applicable, add screenshots to help explain your problem: -->
N/A

## Additional context

<!-- Add any other context about the problem here. -->
No additional context.

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

```console
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.62, NixOS, 25.05 (Warbler), 25.05.20241124.30abd65`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/etc/nixpkgs/channels/nixpkgs`
```

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@zhaofengli
@ovlach
@pinpox

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/or/orca-slicer/package.nix
+++ b/pkgs/by-name/or/orca-slicer/package.nix
@@ -28,6 +28,8 @@ bambu-studio.overrideAttrs (
       )
     '';
 
+    cmakeFlags = lib.remove "-DFLATPAK=1" previousAttrs.cmakeFlags or [ ];
+
     # needed to prevent collisions between the LICENSE.txt files of
     # bambu-studio and orca-slicer.
     postInstall = ''

